packages/contracts-bedrock/src/L1/interfaces/winddown/IBalanceWithdrawer.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/// @title IBalanceWithdrawer
+/// @notice Interface for the BalanceWithdrawer contract
+interface IBalanceWithdrawer {
+    /// @notice Struct for ERC20 balance claim
+    struct Erc20BalanceClaim {
+        address token;
+        uint256 balance;
+    }
+
+    /// @notice Thrown when the caller is not the BalanceClaimer contract
+    error CallerNotBalanceClaimer();
+
+    function balanceClaimer() external view returns (address);
```

Comment:
missing natspec

Reply:
will add the natspec in the contract since it seems to be the practice used in this repo